data/data-pipeline/data_pipeline/ipython/scoring_comparison.ipynb

```diff
@@ -1312,7 +1356,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.6"
+   "version": "3.6.2"
```

Comment:
I am using `pyenv` here and the reference to this version number reflects compatibility for a system-level version. I needed to use `3.6.2` to maintain compatablity with certain spatial statistic packages. From what I understand, this should not interfere with the python kernel.